src/Filter/UnitRefundFilterInterface.php

```diff
@@ -0,0 +1,29 @@
+<?php
+
+/*
+ * This file is part of the Sylius package.
+ *
+ * (c) Paweł Jędrzejewski
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+declare(strict_types=1);
+
+namespace Sylius\RefundPlugin\Filter;
+
+use Sylius\RefundPlugin\Model\UnitRefundInterface;
+
+interface UnitRefundFilterInterface
+{
+    /**
+     * @template T of UnitRefundInterface
+     *
+     * @param T[] $units
```

Comment:
As in the first argument, we will receive "dirty" list of all types of units, but we are saying that it will be `T`. With the current declaration, we are saying that on input, there are the following two arguments, e.g. list of `ShipmentRefund` and the FQCN of ShipmentRefund, and as output, we are returning `ShipmentRefund` again. What is filtrated then? I suppose we should mean, that there will be different types of refunds as the first argument, but the output list will return the classes that are of type `unitRefundClass`

```suggestion
     * @param UnitRefundInterface[] $units
```